packages/webapp/src/components/CodeExamples/Code.js

```diff
@@ -39,21 +38,21 @@ import (
 	"fmt"
 	skynet "github.com/NebulousLabs/go-skynet"
 )
-                        
+
+var client = skynet.New()
+
 func main() {
 	// upload
-	skylink, err := skynet.UploadFile("./src.jpg", skynet.DefaultUploadOptions)
+	skylink, err := client.UploadFile("./src.jpg", skynet.DefaultUploadOptions)
 	if err != nil {
-		fmt.Printf("Unable to upload: %v", err.Error())
-		return
+		panic("Unable to upload: %v", err.Error())
```

Comment:
req: Panic takes a single parameter.

```suggestion
		panic(fmt.Sprintf("Unable to upload: %v", err.Error()))
```